Dockerfile.build

```diff
@@ -0,0 +1,30 @@
+FROM node:11.15.0 AS run
+RUN apt-get update && apt-get install -y python3-pip
+
+
+FROM run AS build
+COPY . /app/
+WORKDIR /app/
+
+RUN npm install
```

Comment:
We should not run npm install. npm ci should be used instead. It is more reliable and makes for a reproducible build.

Reply:
ok